src/main/scala/io/iohk/ethereum/blockchain/sync/FastSync.scala

```diff
@@ -266,10 +276,16 @@ trait FastSync {
       requestedReceipts = requestedReceipts - handler
     }
 
+    private def validateBlocks(requestedHashes: Seq[ByteString], blockBodies: Seq[BlockBody]): Boolean = (requestedHashes zip blockBodies)
```

Comment:
hmm actually maybe not necessarily, as you say we enqueue the remaining blocks in `insertBlocks`